### PR TITLE
Updated docker instructions

### PR DESCRIPTION
**Pull request by Jayclifford345**
Added docker-compose example
Added unzip instructions
Updated folder naming

Closes #

_Describe your proposed changes here._

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-iox/visualize-data/grafana.md [hunk 1]
```diff
@@ -128,18 +128,40 @@ cannot be installed on or used with Grafana Cloud.
...
 ```bash 
 docker run \
-  --volume $PWD/dist:/var/lib/grafana/plugins/influxdata-flightsql-datasource \
+  --volume $PWD/influxdata-flightsql-datasource:/var/lib/grafana/plugins/influxdata-flightsql-datasource \
   --publish 3000:3000 \
   --name grafana \
   --env GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=influxdata-flightsql-datasource \
   grafana/grafana:latest
 ```
+Docker-Compose
```
Comment (sanderson) [inline content/influxdb/cloud-iox/visualize-data/grafana.md:151]:
```suggestion

##### Docker-Compose

```

File: content/influxdb/cloud-iox/visualize-data/grafana.md [hunk 2]
```diff
@@ -128,18 +128,40 @@ cannot be installed on or used with Grafana Cloud.
...
+ The custom plugin directory can exist anywhere in your filesystem as long as
+ the Grafana process can access it.
+
+    ```sh
+    unzip influxdata-flightsql-datasource-0.1.2.zip
+    ```
 
 To add the FlightSQL plugin to your pre-existing Grafana Docker deployment
-mount the following volume to your Grafana container:
+mount the plugin directory `influxdata-flightsql-datasource` as a volume to your Grafana container:
```
Comment (sanderson) [inline content/influxdb/cloud-iox/visualize-data/grafana.md:140]:
```suggestion
mount the plugin directory, `influxdata-flightsql-datasource`, as a volume to your Grafana container:
```

**Review by sanderson — COMMENTED** [review]
@brettbuddin I know you removed these instructions from the original draft of this content. Is there a reason you removed it or any concerns about adding it back in?

**Comment by brettbuddin** [comment]
@sanderson I'll give this some 👀 in the morning.

**Review by sanderson — APPROVED** [review]

**Pull request merged**